### ISSUE TITLE
`space-before-colon` breaks :hover selector for Less file

### ISSUE THREAD

**Issue by svartalf** [csscomb/csscomb.js#280]
Enabled `space-before-colon` breaks less nested selector, note the space between `a:` and `hover`:
```less
/* before */

li {
	a, a:link, a:visited, a:hover {
		font-weight: normal;
		text-decoration: underline;
	}
	a:hover {
		text-decoration: none;
	}
}

/* after */

li {
    a: hover {
        text-decoration: none;
    }
        a, a:link, a:visited, a:hover {
            font-weight: normal;

            text-decoration: underline;
        }
}
```

csscomb version: 3.0.0

**Comment by SpencerCarstens** [comment]
Best bet would be to create a "whitelist" of pseudo-classes to not insert the space for.

Such as these: https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes

also similar ticket --> https://github.com/csscomb/csscomb.js/issues/309

**Comment by apexskier** [comment]
Looks like this is a result of a bug in the CSS parser (tonyganch/gonzales-pe#28). It's not parsing sub-rulesets in less properly, but is actually parsing them as declarations.

**Comment by apexskier** [comment]
Upgrading gonzales-pe would probably fix it, but I think it requires changing a lot of code.

**Comment by tonyganch** [comment]
Fixed in `dev`, published as `@4.0.0-alpha`.
